cms_coding_rules.md

```diff
@@ -34,7 +34,7 @@ Coding rules are meant to prevent serious problems in software function, perform
 10. For a getter method, using the value name is preferred, e.g. `momentum()`.
 11. Do not use single character names, except for loop indices.
 12. Do not use special characters, except for "_" where allowed.
-13. Do not use "\_" as ﬁrst character. Only use it as the last character for class data member names, not local variable names.
+13. Do not use "\_" as ﬁrst character, except for user-defined literals which are required to start with "\_". Only use it as the last character for class data member names, not local variable names.
```

Comment:
shouldn't this refer only to the user-defined suffixes used for the user-defined literals?

An example from https://en.cppreference.com/w/cpp/language/user_literal
``` 1-4) user-defined integer literals, such as 12_km ```

Reply:
Thanks for pointing out the lack of clarity. I revised the line.